src/components/Background/index.css

```diff
@@ -25,15 +25,28 @@
   }
 }
 
-.image {
+.stretch-to-fit {
   position: absolute !important;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  top: -50%;
+  left: -50%;
 
-  width: 100%;
-  height: 100%;
+  width: 200%;
+  height: 200%;
```

Comment:
Please rebase with master, I think this is stretch too much the video, but I want to see with the new videos.

Reply:
It's a hack for Edge and IE, no other way to do it.

Reply:
Ah ok, only if object-fit not supported